projects/fal/pyproject.toml

```diff
@@ -41,7 +40,6 @@ types-python-dateutil = "^2.8.0"
 
 # For 3.9 and earlier, importlib-metadata's newer versions are included in the standard library.
 importlib-metadata = { version = ">=4.4", python = "<3.10" }
-boto3 = "^1.33.8"
```

Comment:
this is only used in r2 implementation, and we don't really like to bring it down with us (just like google-cloud-storage). cc: @badayvedat

Reply:
makes sense! let's remove it